certified-connectors/OpenLegacy IBM I (AS400)/apiDefinition.swagger.json

```diff
@@ -1,318 +1,477 @@
+                      "type": "boolean",
+                      "in": "query",
+                      "default": true,
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines that Contract ID must be included in the response.",
+                      "description": "Determines that Contract ID must be included in the response.",
+                      "required": true
+                  },
+                  {
+                      "name": "module_type",
+                      "type": "string",
+                      "in": "query",
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines the BackendSolution of module by which projects must be filtered.",
+                      "description": "Determines the BackendSolution of module by which projects must be filtered.",
+                      "required": true
+                  }
+              ],
+              "description": "Returns a list of user's projects",
+              "responses": {
+                  "200": {
+                      "description": "default",
+                      "schema": {
+                          "$ref": "#/definitions/GetAllProjectsResponse"
+                      }
+                  }
+              }
+          }
+      },
+      "/api/v1/methods": {
+          "get": {
+              "summary": "Get Methods for Contract",
+              "operationId": "GetMethodsForContract",
+              "description": "Get a list of methods by Contract ID",
+              "parameters": [
+                  {
+                      "name": "contractId",
+                      "in": "query",
+                      "required": true,
+                      "type": "string",
+                      "x-ms-summary": "The Contract ID selected by user.",
+                      "description": "The Contract ID selected by user."
+                  },
+                  {
+                      "name": "enrichmentType",
+                      "in": "query",
+                      "default": "HTTP",
+                      "type": "string",
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines the type of enrichment metadata to be included in the response.",
+                      "description": "Determines the type of enrichment metadata to be included in the response.",
+                      "required": true
+                  },
+                  {
+                      "name": "published",
+                      "type": "boolean",
+                      "in": "query",
+                      "default": true,
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines that Published method only will be included in the response.",
+                      "description": "Determines that Published method only will be included in the response.",
+                      "required": true
```

Comment:
@boiv The required parameter has been added. can you please confirm this change?

Reply:
Yes, this is a new parameter which is required